### PR TITLE
docs: translate chinese doc of faq.md and monitoring.md into english (#1796)

### LINKED ISSUE #1794: docs: translate the Chinese doc of monitoring.md into English
1. The source of the Chinese doc is here: https://github.com/4paradigm/OpenMLDB/blob/main/docs/zh/maintain/monitoring.md
2. The translated English doc should be saved under the path of docs/en/maintain/monitoring.md
3. Add the page entry to the file docs/en/maintain/index.rst

### LINKED ISSUE #1796: docs: translate the Chinese doc of faq.md into English
1. The source of the Chinese doc is here: https://github.com/4paradigm/OpenMLDB/blob/main/docs/zh/maintain/faq.md
2. The translated English doc should be saved under the path of docs/en/maintain/faq.md
3. Add the page entry to the file docs/en/maintain/index.rst

### PR DESCRIPTION
**Pull request by jmoldyvan**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
-make a new faq.md file in the docs/en/maintain folder
-add faq to index rst
close https://github.com/4paradigm/OpenMLDB/issues/1796
close #1794 

* **What is the current behavior?** (You can also link to an open issue here)
-an faq docs listing some questions

* **What is the new behavior (if this is a feature change)?**
-translation from chinese faq
-add new file
-add faq to indexrst

### CODE REVIEW

File: docs/en/maintain/faq.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Operation and maintenance FAQ
```
Comment (lumianph) [inline docs/en/maintain/faq.md]: maintenance => Maintenance 
(capitalize the titles)

File: docs/en/maintain/faq.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+## Deploy and start FAQ
```
Comment (lumianph) [inline docs/en/maintain/faq.md]: ## Deployment and Startup FAQ

File: docs/en/maintain/faq.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+### 1. How to confirm that the cluster is running normally？
+Although there is one-click to start the script, due to the numerous configurations, problems such as "the port is occupied" and "the directory does not have read and write permissions" may occur. These problems can only be discovered after the server process is running, and there is no timely feedback after exiting. (If monitoring is configured, it can be checked directly by monitoring.）
```
Comment (lumianph) [inline docs/en/maintain/faq.md]: discovered => identified
after => when

File: docs/en/maintain/faq.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+Therefore, please make sure that all server processes in the cluster are running normally.
+
+It can be queried by `ps axu | grep openmldb`. (Note that `mon` is used as the daemon process in the official run script, but the running of the `mon` process does not mean that the openmldb server process is running.)
```
Comment (lumianph) [inline docs/en/maintain/faq.md]: openmldb => OpenMLDB

File: docs/en/maintain/faq.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+
+If the processes are all running and the cluster still behaves abnormally, you need to query the server log. You can give priority to 'WARN' and 'ERROR' level logs, which are most likely the root cause.
+
+## Server FAQ
+
+### 1. Why is there a Fail to write into Socket warning in the log?
```
Comment (lumianph) [inline docs/en/maintain/faq.md]: Why is there a warning of "Failed to write into Socket ..." in the log?

**Review by lumianph — CHANGES_REQUESTED** [review]
@jmoldyvan thank you for your contribution! Are you interested in making more contribution to this project? You may join our slack channel to talk to me :-) 
https://join.slack.com/t/openmldb/shared_invite/zt-ozu3llie-K~hn9Ss1GZcFW2~K_L5sMg

**Comment by jmoldyvan** [comment]
Do you want me to make these changes and resubmit a PR with them?

**Comment by lumianph** [comment]
> 

no, don't resubmit a PR, just push another commit to this PR to address those comments.

**Comment by jmoldyvan** [comment]
I also committed:

docs: translate the Chinese doc of monitoring.md into English #1794

What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)
-make a new faq.md file in the docs/en/maintain folder for monitoring
-add monitoring to index rst
close https://github.com/4paradigm/OpenMLDB/issues/1796
close docs: translate the Chinese doc of monitoring.md into English #1794

What is the current behavior? (You can also link to an open issue here)
-an overview of monitoring procedures

What is the new behavior (if this is a feature change)?
-translation from chinese monitoring.md
-added new file to docs/en/maintain/monitoring.md
-add monitoring to indexrst

File: docs/en/maintain/monitoring.md [hunk 6]
```diff
@@ -0,0 +1,200 @@
...
+## Overview
+
+
+The monitoring scheme of OpenMLDB is outlined as follows：
+
+- Use [prometheus](https://prometheus.io) to collect monitoring metrics, [grafana](https://grafana.com/oss/grafana/) to visualize metrics
+- OpenMLDB exporter exposes database-level and component-level monitoring metrics
+- Node uses [node_exporter](https://github.com/prometheus/node_exporter) to expose machine and operating system related metrics
+
+## Install and run OpenMLDB exporter
```
Comment (lumianph) [inline docs/en/maintain/monitoring.md]: Install and Run OpenMLDB Exporter

File: docs/en/maintain/monitoring.md [hunk 7]
```diff
@@ -0,0 +1,200 @@
...
+
+### Introduction
+
+[![PyPI](https://img.shields.io/pypi/v/openmldb-exporter?label=openmldb-exporter)](https://pypi.org/project/openmldb-exporter/)
+![PyPI - Python Version](https://img.shields.io/pypi/pyversions/openmldb-exporter?style=flat-square)
+
+The OpenMLDB exporter is a prometheus exporter implemented in Python. The core connects the OpenMLDB instance through the database SDK and will query the exposed monitoring indicators through SQL statements. Exporter will follow the OpenMLDB version update and release to PyPI. For production use, you can install the latest `openmldb-exporter` directly through pip. For development and usage instructions, please refer to the code directory [README](https://github.com/4paradigm/OpenMLDB/tree /main/monitoring).
```
Comment (lumianph) [inline docs/en/maintain/monitoring.md]: https://github.com/4paradigm/OpenMLDB/tree /main/monitoring
there is a space between "tree" and "/main", which will cause an invalid URL

File: docs/en/maintain/monitoring.md [hunk 8]
```diff
@@ -0,0 +1,200 @@
...
+
+2. Start OpenMLDB
+
+    See [install_deploy](../deploy/install_deploy.md) How to start OpenMLDB components
+
+    OpenMLDB exporter requires OpenMLDB to start the server status function, to do so, add the startup parameter `--enable_status_service=true` at startup, please make sure that `--enable_status_service=true' in `conf/(tablet|nameserver).flags` in the installation directory `.
+
+   The default startup script `bin/start.sh` enables server status, no additional configuration is required.
+   
+3. Note: Make sure to select the binding IP addresses of OpenMLDB components,       OpenMLDB exporter as well as prometheus and grafana to ensure that grafana can access prometheus, and that prometheus, OpenMLDB exporter, and OpenMLDB components can access each other.
```
Comment (lumianph) [inline docs/en/maintain/monitoring.md]: "components,       OpenMLDB"
remove the extra spaces between "components," and "OpenMLDB" (it seems a tab)

File: docs/en/maintain/monitoring.md [hunk 9]
```diff
@@ -0,0 +1,200 @@
...
+      openmldb_connected_seconds_created{endpoint="172.17.0.15:9520",role="tablet"} 1.6501813860467942e+09                   
+      openmldb_connected_seconds_created{endpoint="172.17.0.15:9521",role="tablet"} 1.6501813860495396e+09                   
+      openmldb_connected_seconds_created{endpoint="172.17.0.15:9522",role="tablet"} 1.650181386050323e+09                    
+      openmldb_connected_seconds_created{endpoint="172.17.0.15:9622",role="nameserver"} 1.6501813860512116e+09               
+      openmldb_connected_seconds_created{endpoint="172.17.0.15:9623",role="nameserver"} 1.650181386051238e+09                
+      openmldb_connected_seconds_created{endpoint="172.17.0.15:9624",role="nameserver"} 1.6501813860512598e+09               
+      
+   ```
+
+## Deploy node exporter
```
Comment (lumianph) [inline docs/en/maintain/monitoring.md]: Deploy Node Exporter

File: docs/en/maintain/monitoring.md [hunk 10]
```diff
@@ -0,0 +1,200 @@
...
+
+For installation and deployment of prometheus and grafana, please refer to the official documents [promtheus get started](https://prometheus.io/docs/prometheus/latest/getting_started/) and [grafana get started](https://grafana.com/docs/ grafana/latest/getting-started/getting-started-prometheus/).
+OpenMLDB provides prometheus and grafana configuration files for reference, see [OpenMLDB mixin](https://github.com/4paradigm/OpenMLDB/tree/main/monitoring/openmldb_mixin/README.md)
+
+- prometheus_example.yml: Prometheus configuration example, intended to modify the target address in `node`, `openmldb_components` and `openmldb_exporter` jobs
+- openmldb_dashboard.json: grafana dashboard configuration for OpenMLDB metrics, divided into two steps:
+   1. Under the grafana data source page, add the started prometheus server address as the data source
+   2. Under the dashboard browsing page, click New to import a dashboard, and upload the json configuration file
+
+## Understand existing monitoring metrics
```
Comment (lumianph) [inline docs/en/maintain/monitoring.md]: Understand Existing Monitoring Metrics

File: docs/en/maintain/monitoring.md [hunk 11]
```diff
@@ -0,0 +1,200 @@
...
+   - deploy query response time: the runtime of the deployment query inside the tablet
+
+   accessible
+
+   ```bash
+   curl http://172.17.0.15:8000/metrics
+   ```
+
+   curl http://172.17.0.15:8000/metrics
+    View full DB-Level metrics and help information.
```
Comment (lumianph) [inline docs/en/maintain/monitoring.md]: remove this line

File: docs/en/maintain/monitoring.md [hunk 12]
```diff
@@ -0,0 +1,200 @@
...
+
+   The categories of indicators exposed are mainly:
+
+   - component status: cluster component status
+
+   - table status: database table related information, such as `rows_cout`, `memory_bytes`
+
+   - deploy query response time: the runtime of the deployment query inside the tablet
+
+   accessible
```
Comment (lumianph) [inline docs/en/maintain/monitoring.md]: The full DB-Level metrics can be listed through the following command:

(sorry the Chinese doc is also not good)

File: docs/en/maintain/monitoring.md [hunk 13]
```diff
@@ -0,0 +1,200 @@
...
+
+   ```bash
+   curl http://172.17.0.15:8000/metrics
+   ```
+
+   curl http://172.17.0.15:8000/metrics
```
Comment (lumianph) [inline docs/en/maintain/monitoring.md]: remove this line

File: docs/en/maintain/monitoring.md [hunk 14]
```diff
@@ -0,0 +1,200 @@
...
+    View full DB-Level metrics and help information.
+
+2. Component-Level indicator. The related components of OpenMLDB (nameserver, tablet, etc), themselves as BRPC server, and expose [prometheus related metrics](https://github.com/apache/incubator-brpc/blob/master/docs/en/bvar .md#export-to-prometheus), you only need to configure the prometheus server to pull metrics from the corresponding address. Corresponding to the `job_name=openmldb_components` item in `prometheus_example.yml`:
```
Comment (lumianph) [inline docs/en/maintain/monitoring.md]: indicator => metrics (let's use the same term)
Corresponding to (It corresponds to ...)

File: docs/en/maintain/monitoring.md [hunk 15]
```diff
@@ -0,0 +1,200 @@
...
+       # job to pull component metrics from OpenMLDB like tablet/nameserver
+       # tweak the 'targets' list in 'static_configs' on your need
+       # every nameserver/tablet component endpoint should be added into targets
+       metrics_path: /brpc_metrics
+       static_configs:
+         - targets:
+           - 172.17.0.15:9622
+   ```
+
+   The indicators of exposure are mainly
```
Comment (lumianph) [inline docs/en/maintain/monitoring.md]: indicators => metrics

File: docs/en/maintain/monitoring.md [hunk 16]
```diff
@@ -0,0 +1,200 @@
...
+
+   - BRPC server process related information
+   - Corresponding to the RPC method related indicators defined by the BRPC server, such as the RPC request `count`, `error_count`, `qps` and `response_time`
```
Comment (lumianph) [inline docs/en/maintain/monitoring.md]: indicators => metrics

File: docs/en/maintain/monitoring.md [hunk 17]
```diff
@@ -0,0 +1,200 @@
...
+
+   pass
```
Comment (lumianph) [inline docs/en/maintain/monitoring.md]: Metrics and help information can be shown through the following command (Note that the metrics exposed by different components will vary):

File: docs/en/maintain/monitoring.md [hunk 18]
```diff
@@ -0,0 +1,200 @@
...
+
+   ```bash
+   curl http://${COMPONENT_IP}:${COMPONENT_PORT}/brpc_metrics
+   ```
+
+   View metrics and help information. Note that the metrics exposed by different components will vary.
```
Comment (lumianph) [inline docs/en/maintain/monitoring.md]: remove this line

**Review by lumianph — CHANGES_REQUESTED** [review]
thank you for the contribution! Please help address the comments.
BTW, if you have another translated doc, please submit another PR

**Review by lumianph — APPROVED** [review]

**Pull request merged**